<launch>
	<node pkg="robot_base" type="robot_base_node" name="robot_base_node" output="screen">
		<param name="serial_port" type="string" value="/dev/ttyUSB0"/>
        <param name="serial_port" type="string" value="115200"/>
	</node>
</launch>